pyfuncol/tests/test_no_forbiddenfruit.py

```diff
@@ -0,0 +1,7 @@
+from pyfuncol.list import *
+import pytest
+
+my_list = [1, 2, 3]
+
```

Comment:
I would add two additional tests, one for dict and one for set, mainly for completion.

Reply:
[Tests added](https://github.com/m-doescode/pyfuncol/commit/c5820b599b8333efcaa8e30530b6abe8738e96f5)